presto-spark-base/src/test/java/com/facebook/presto/spark/execution/http/TestPrestoSparkHttpClient.java

```diff
@@ -678,13 +682,14 @@ public void testResultFetcherTransportErrorFail()
     }
 
     @Test
+    @Ignore
```

Comment:
why did you disable this test? Add ac comment and link to an issue. Also, use `@Test(enabled=false)` instead of `@ignore`